situp-core/src/main/java/com/amazon/situp/parser/PipelineParser.java

```diff
@@ -125,6 +141,7 @@ private int getDefaultProcessorThreads() {
     }
 
     private Sink buildSinkOrConnector(final PluginSetting pluginSetting) {
+        LOG.info("Building [{}] as sink component", pluginSetting.getName());
         final Optional<String> pipelineNameOptional = getPipelineNameIfPipelineType(pluginSetting);
         if (pipelineNameOptional.isPresent()) { //update to ifPresentOrElse when using JDK9
```

Comment:
Same as above.

Reply:
if the configured sink is of type pipeline, then it will return pipeline name; if it is not of pipeline type, then we won't have name. so i think we need Optional.